packages/xrpl/test/shamap.test.ts

```diff
@@ -1,6 +1,6 @@
 import assert from 'assert'
 
-import SHAMap, { NodeType } from '../src/utils/hashes/SHAMap'
+import SHAMap, { NodeType } from 'xrpl-local/utils/hashes/SHAMap'
```

Comment:
This seems like it's going backwards in readability/usability, especially if we're trying to get rid of the use of xrpl-local

Reply:
Agree, fixed